js/contextMenus.js

```diff
@@ -376,6 +378,16 @@ const siteSingleDetailTemplate = (siteKey, type, activeFrame) => {
     )
   }
 
+  if (type === siteTags.HISTORY || type === siteTags.BOOKMARK) {
```

Comment:
should this also check that payments is enabled?

Reply:
Very good catch, will update

Reply:
updated